tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -224,7 +231,7 @@ install_cc_shimv2() {
 	export REMOVE_VMM_CONFIGS="acrn fc"
 
         extra_opts="DEFSERVICEOFFLOAD=true"
-	if [ -f "${repo_root_dir}/tools/osbuilder/root_hash.txt" ]; then
+	if [ -f "${repo_root_dir}/tools/osbuilder/root_hash_vanilla.txt" ]; then
```

Comment:
With the new tdx root_hash suffix, should there be an if to check for `"${repo_root_dir}/tools/osbuilder/root_hash_tdx.txt"` as well?

Reply:
So this was answered in the next commit, but maybe we could considering squashing them together so the commits work better. Definitely an optional nit though!